hawc/apps/riskofbias/api.py

```diff
@@ -224,11 +224,10 @@ def create_v2(self, request):
 
     @action(detail=True, methods=("patch",))
     def update_v2(self, request, *args, **kwargs):
-        partial = kwargs.pop("partial", False)
```

Comment:
This is leftover from the default `update` method. Since this is a `PATCH` action and not a `PUT`, though, we can assume that `partial=True`.